modules/op-release-notes-1-8.adoc

```diff
@@ -44,8 +44,29 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+[id="fixed-issues-1-8_{context}"]
+== Fixed issues
 
-// Pipelines
-// *
+* With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even if GitHub Apps is not feasible.
+// (link:https://issues.redhat.com//browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+* With this update, as a developer, you can use the *Developer* perspective of the {product-title} web console to run all the `PipelineRun` resources in your `.tekton` directory that match specified events. The console displays the status and reports of the runs. If you are using WebVCS, the reports display comments.
+// (link:https://issues.redhat.com//browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+* With this update, as an administrator, you can install Tekton Hub with minimal input because it has better default configuration settings. You can start running Tekton Hub and work with your teams to decide what customizations they need.
+// (link:https://issues.redhat.com//browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+* With this update, as a developer, the `tkn-pac` CLI tool notifies you if you create a duplicate repository to a {pac} runs. When you use`tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking executions.
+// (link:https://issues.redhat.com//browse/SRVKP-1758[SRVKP-1758 Pipeline as code: validate repository URL uniqueness])
+// Shivam Mukhade
+
+* With this update, as an administrator, you can configure Tekton Hub to use a custom database rather than the default database.
+// (link:https://issues.redhat.com//browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* With this update, as an administrator, if you enable Tekton Hub, it periodically updates the catalog with the current information. You can configure the length of this period.
```

Comment:
Previously, we used to have cronjob which used to update TektonHub Database for any new changes in Catalog.
now with this feature, it is inbuilt in Tekton Hub
after a certain TektonHub will sync any new changes in Catalog and update the database and then show on the UI
Example:
resouces will shown on https://hub.tekton.dev/ if a new resources is added in https://github.com/tektoncd/catalog
it will itself sync the new changes.
 user can configure the length of this period.

Reply:
@sm43 Thanks for this explanation. I'm a bit confused about what we're describing. I'll need to talk through this a bit with you. Perhaps on Slack.

Reply:
@sm43 @PuneetPunamiya, I believe this enhancement affects the local Tekton Hub instance, if enabled, not https://hub.tekton.dev/

Reply:
@sm43 Updated. Please review and approve. Thanks.